src/client/app/app-routing.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  imports: [
+    RouterModule.forRoot([
+      /* define app non-feature routes here */
```

Comment:
Ah thanks for the doc links! Hehe, yeah specifically with the 'new' router it is often kind of hard to find the most up-to-date info :)